about.html

```diff
@@ -0,0 +1,153 @@
+        </ul>
+    </div>
+    </section>
+    
+    <section class="see container-fluid text-center text-dark bg-white">
+        <div class="row pt-3 mb-5">
+          <div class="col-12">
+            <h4 class="text-center about-section-title">See the previous Symposium.</h4>
+            <hr class="hr-title">
+            <p class="about-info">Take a look at the last two datacamp webinars which took place<br>
+                 in Nairobi Kenya and Kampala Uganda.</p>
+          </div>
+        </div>
+    </section>
+        <div class="pst text-center" >
+            <div class="about-pst">
+                <div class="about-past">
+                <h3>2021</h3>
+                <p class="about-past-one">Interior design symposium 2021 kampala.</p>
+            </div>
+                <div class="about-pst-one">
+                    <div class="about-past">
+                    <h3>2020</h3>
+                        <p class="about-past-one">Interior design symposium 2021 Nairobi.</p>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+    
+        <section class="partners-section">
+            <h4 class="partners-heading"><strong>Partners</strong></h4>
+            <hr />
+            <div class="partners-flex">
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/main/images/partner1.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner2.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner3.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner4.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner5.png" alt="" />
+              </div>
+            </div>
+          </section>
+         <footer class="container-fluid p-5 bg-light">
+            <div class="row d-flex justify-content-center">
+              <div class="col-4 d-flex justify-content-center">
+                 <h2 class="nav-logo">SUAVE INTERIORS</h2>
+              </div>
+              <div class="col-4 d-flex justify-content-center align-items-center">
+                <p>2022 Creative Commons Korea <br>
+                Some Rights Reserved </p>
+              </div>
+            </div>
+          </footer>
```

Comment:
- [ ] Kindly make sure that the pages  look almost identical to the original design. For Desktop size In the About page there is no partner section just like design show below and for footer section Adjust  background color just like Design Template.

The Design About Page Look Likes this : 
 
![C1](https://user-images.githubusercontent.com/93598189/191454318-5104d08c-22f5-485a-b25d-a4de10e4158d.PNG)

Your Project About Page Look Likes this : 
![C2](https://user-images.githubusercontent.com/93598189/191454415-3216a3b6-c252-4f9c-a8db-66e0e3e2cc3a.PNG)